packages/store/src/hypernative/hypernativeApi.dto.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  HypernativeAssessmentData,
+  HypernativeBalanceChanges,
+  HypernativeFinding,
+  HypernativeRiskSeverity,
+  HypernativeTx,
+} from '@safe-global/utils/features/safe-shield/types/hypernative.type'
+
+export type HypernativeAssessmentRequestDto = {
+  safeAddress: `0x${string}`
+  safeTxHash: `0x${string}`
+  transaction: HypernativeTx
+  url: string // @Todo: Can we make it optional?
+}
+
+export type HypernativeRiskDto = {
+  title: string
+  details: string
+  severity: HypernativeRiskSeverity
+}
+
+export type HypernativeAssessmentResponseDto = {
+  safeTxHash: `0x${string}`
+  status: 'OK' | 'ERROR'
```

Comment:
We seem to have two error statuses now: "ERROR" and "FAILED". But in the current API doc I can only see "FAILED". So I think we should remove the "ERROR" option here.
Also we could acutally rename the DTO to make it clearer that this one is only for the success state. Maybe `HypernativeAssessmentSuccessResponseDto`?
```suggestion
  status: 'OK'
```